samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/dashboard/DashboardViewModel.kt

```diff
@@ -40,4 +41,23 @@ class DashboardViewModel @Inject constructor(
             onLogout.invoke()
         }
     }
+
+    fun sendRandomEvent() {
+        when (Random().nextInt(3)) {
+            0 -> {
+                CustomerIO.instance().track("Order Purchased")
+            }
+
+            1 -> {
+                val attributes = mapOf("movie_name" to "The Incredibles")
+                CustomerIO.instance().track("movie_watched", attributes)
+            }
+
+            2 -> {
+                val sevenDaysLater = System.currentTimeMillis() / 1000 + 7 * 24 * 60 * 60
```

Comment:
At a minimum, could we have a comment added to this line to explain the logic behind it?

Reply:
I am finding it hard what to add as a comment here. I felt variable name `sevenDaysLater` explains its a timestap after 7 days? 

and why 7 days? it's because its the requirement for the sample apps "random events"

Reply:
The variable explains it perfectly. 

My suggestion is around the logic, `/ 1000 + 7 * 24 * 60 * 60` with a comment helping explain what each of those integers are doing. My thought is that by explaining each of the numbers, it would help make the code more maintainable in the future. 

Another idea is using Android's Calendar API. 

```kt
Calendar.getInstance().apply { add(Calendar.DAY_OF_YEAR, 7) }.time
```